### DISCUSSION TITLE
Airplane starts rocking side to side in cruise

### DISCUSSION THREAD

**Discussion by FippeH** [flybywiresim/aircraft#2353] [Help]
Hello,

I've got a problem with the mod (I think)

When I'm in cruise the airplane starts rocking hard side to side. 

I dont know if I have done anything wrong with the setup of the plane or if it's a bug.

If I disable the autopilot it still does the same thing.

I uploaded a video on youtube to show what I mean. (https://www.youtube.com/watch?v=wQqOImeI6Eo&feature=youtu.be)

Thanks in advance

**Comment by Benjozork** [comment]
Hello, this is fixed in the development version. Can you try that and report back ?

**Reply by FippeH** [reply]
Yup, seems to work.

Thanks :)

**Comment by jkennebeck** [comment]
Also, once about about 20000 ft if you disable SEC1 in the upper panel it won't roll in cruise or climb.  Upper panel left side under Flight Control systems panel.  FLT CTL middle push button.
![Capture](https://user-images.githubusercontent.com/34457862/101947211-15c52a00-3bbe-11eb-9467-ddcbefceb3ab.JPG)

**Reply by Benjozork** [reply]
This is fixed in the development version, no need for a workaround
